layers/meta-balena-raspberrypi/recipes-bsp/rpi-eeprom/files/pieeprom-update.sh

```diff
@@ -49,7 +49,6 @@ fi
 spi_bind unbind
 
 /usr/bin/vcmailbox 0x00030056 4 4 0 > /dev/null || true
-/usr/bin/dtparam -d /mnt/boot/overlays/ audio=off
```

Comment:
Why not apply & unapply the `audremap` overlay here? The way it is will only work on newly provisioned devices, existing devices won't pick up the new overlay in `config.txt`

Reply:
You're right - the advantage of `config.txt` is that audio wouldn't get interrupted on EEPROM update, but it's a problem for existing devices as you point out.

I'll re-work it.